.gitignore

```diff
@@ -1,12 +1,14 @@
 */.DS_Store
 .DS_Store
+build/
```

Comment:
making sure the gitignore file(s) are setup properly is definitely something I'd love another set of eyes on.  Things seem to working well now, but they can probably be cleaned up.